app/javascript/react/src/containers/SearchResultsContainer.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import RestaurantTile from '../components/RestaurantTile'
+
+class SearchResultsContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    }
+  }
+  render() {
+    let restaurants = this.props.restaurants.map (restaurant => {
+      return(
+        <RestaurantTile
+          key={restaurant.id}
+          name={restaurant.name}
+          address={restaurant.address}
+          city={restaurant.city}
+          state={restaurant.state}
+          zip={restaurant.zip}
+          dollar_value={restaurant.dollar_value}
+          description={restaurant.description}
+          picture={restaurant.picture}
+          phone_number={restaurant.phone_number}
+          website_url={restaurant.website_url}
+          dining_type={restaurant.restaurant_dining_type}
+          food_type={restaurant.restaurant_food_type}
```

Comment:
These props are a little long for my taste. Why not simply pass down `restaurant={restaurant}`, and then decompose in the presentational component instead?